### PR TITLE
Change wording form layered to cross-stack references

### PR DESCRIPTION
**Pull request by rezabekf**
## What does this PR do and why?

This PR changes the wording to be consistent with AWS  terminology.

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] Added/Updated documentation if applicable.
- [x] Pre-commit checks passed.
- [x] Lint and Nag checks passed.
- [x] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/intermediate/templates/cross-stacks/index.md [hunk 1]
```diff
@@ -72,26 +70,26 @@ Outputs:
 ##### 2. Deploy the VPC Stack
 :::::tabs{variant="container"}
 ::::tab{id="cloud9" label="Cloud9"}
-1. In the **Cloud9 terminal** navigate to `cfn101-workshop/code/workspace/layered-stacks`.
+1. In the **Cloud9 terminal** navigate to `cfn101-workshop/code/workspace/cross-stacks`.
 1. **Create Stack** by using the following AWS CLI command. The template requires you provide the values for `AvailabilityZones` parameter, For example `us-east-1a` and `us-east-1b` are used below. Please select 2 Availability Zone based on your region.
 :::code{language=shell showLineNumbers=false showCopyAction=true}
 aws cloudformation create-stack \
---stack-name cfn-workshop-layered-stacks-vpc \
+--stack-name cfn-workshop-cross-stackss-vpc \
```
Comment (mrinaudo-aws) [inline content/intermediate/templates/cross-stacks/index.md:77]:
```suggestion
--stack-name cfn-workshop-cross-stacks-vpc \
```

**Review by mrinaudo-aws — CHANGES_REQUESTED** [review]
Thank you @rezabekf - please fix the typo to which I am pointing in my suggestion, otherwise LGTM.

**Comment by rezabekf** [comment]
@cmick-aws, could you please review Japanese translation for this change? I have updated static files links, however the text will need to be done by you or your team. Thank you!

**Review by mrinaudo-aws — APPROVED** [review]
LGTM

**Comment by cmick-aws** [comment]
@rezabekf , Please let me apply corresponding changes to the other PRs.

**Pull request merged**